src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -3,184 +3,245 @@
+import { WorkspaceUseCaseCard } from './workspace_use_case_card';
+import { WorkspaceUseCaseFlyout } from '../workspace_form';
+import { navigateToWorkspacePageWithUseCase } from '../utils/workspace';
 
-export const WorkspaceInitial = () => {
+export interface WorkspaceInitialProps {
+  registeredUseCases$: BehaviorSubject<WorkspaceUseCase[]>;
+}
+
+export const WorkspaceInitial = ({ registeredUseCases$ }: WorkspaceInitialProps) => {
   const {
-    services: { application, chrome, uiSettings },
+    services: { application, chrome, workspaces, http },
   } = useOpenSearchDashboards<CoreStart>();
-  const isDashboardAdmin = application.capabilities.dashboards?.isDashboardAdmin;
-  const logos = chrome.logos;
-  const createWorkspaceUrl = application.getUrlForApp(WORKSPACE_CREATE_APP_ID, { absolute: true });
-  const settingsAndSetupUrl = application.getUrlForApp('settings_landing', { absolute: true });
-  const isDarkTheme = uiSettings.get('theme:darkMode');
-  const backGroundUrl = isDarkTheme ? BackgroundDarkSVG : BackgroundLightSVG;
+  const isDashboardAdmin = !!application.capabilities.dashboards?.isDashboardAdmin;
+  const availableUseCases = registeredUseCases$
+    .getValue()
+    .filter((item) => !item.systematic || item.id === 'all');
+  const workspaceList = workspaces.workspaceList$.getValue();
+  const [isUseCaseFlyoutVisible, setIsUseCaseFlyoutVisible] = useState(false);
+  const [defaultUseCaseId, setDefaultUseCaseId] = useState(availableUseCases[0].id);
+
+  const handleClickUseCaseInformation = useCallback((useCaseId: string) => {
+    setIsUseCaseFlyoutVisible(true);
+    setDefaultUseCaseId(useCaseId);
+  }, []);
+  const handleFlyoutClose = useCallback(() => {
+    setIsUseCaseFlyoutVisible(false);
+  }, []);
+
+  const useCaseCards = availableUseCases.map((useCase) => {
+    return (
+      <EuiFlexItem key={useCase.id}>
+        <WorkspaceUseCaseCard
+          useCase={useCase}
+          workspaces={workspaceList}
+          application={application}
+          http={http}
+          isDashboardAdmin={isDashboardAdmin}
+          handleClickUseCaseInformation={handleClickUseCaseInformation}
+        />
+      </EuiFlexItem>
+    );
+  });
+
+  const mountUserAccountRef = useRef<HTMLDivElement>(null);
+  const mountSettingRef = useRef<HTMLDivElement>(null);
+  const mountDevToolsRef = useRef<HTMLDivElement>(null);
+
+  const [isCreateWorkspacePopoverOpen, setIsCreateWorkspacePopoverOpen] = useState(false);
+  const [userAccountMount, setUserAccountMount] = useState<ChromeNavControl | undefined>(undefined);
+  const [settingMount, setSettingMount] = useState<ChromeNavControl | undefined>(undefined);
+  const [devToolsMount, setDevToolsMount] = useState<ChromeNavControl | undefined>(undefined);
+
+  useEffect(() => {
+    const subscription = chrome.navControls.getLeftBottom$().subscribe((items) => {
+      setSettingMount(items.at(2));
+      setDevToolsMount(items.at(3));
+      setUserAccountMount(items.at(-1));
```

Comment:
Can we use id to match instead picking from fixed index?

Reply:
Currently, The data structure passed into `registerLeftBottom()` is 
```
export interface ChromeNavControl {
  order?: number;
  mount: MountPoint;
}
```
and there is no `id` .
In the future, we can increase the robustness of the code by adding an optional `id`, and we can pick the it by matching `id`.

Reply:
I see, but things will be super tricky that security plugin may not be installed. Can we pick by `order`?

Reply:
Yes, I will pick mount by order